packages/assets/package.json

```diff
@@ -14,6 +14,18 @@
   "main": "dist/cjs/assets.js",
   "module": "dist/esm/assets.js",
   "bundle": "dist/browser/assets.js",
+  "exports": {
```

Comment:
```suggestion
  "types": "index.d.ts",
  "exports": {
```

Reply:
`types` was missing too, adding for backward compatibility

Reply:
okay

Reply:
Could you commit this change please?